pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1805,6 +1805,13 @@ public static class ForwardIndexConfigs {
         "pinot.forward.index.default.target.docs.per.chunk";
   }
 
+  public static class FieldSpecConfigs {
+    public static final String CONFIG_OF_DEFAULT_JSON_SANITIZATION_STRATEGY =
```

Comment:
Keep the name consistent with the config key: `pinot.field.spec.default.json.max.length.exceed.strategy`